pkgs/tools/audio/nanotts/default.nix

```diff
@@ -0,0 +1,35 @@
+
+{ lib, stdenv, fetchFromGitHub, autoconf, automake, libtool, popt, alsaLib }:
+
+stdenv.mkDerivation {
+  pname = "nano-tts";
+  version = "unstable-2021-2-22";
+
+  src = fetchFromGitHub {
+    repo = "nanotts";
+    owner = "gmn";
+    rev = "d8b91f3d9d524c30f6fe8098ea7a0a638c889cf9";
+    sha256 = "sha256-bFu3U50zc90iQeWkqOsCipkueJUZI3cW5342jjYSnGI=";
+  };
+  nativeBuildInputs = [ autoconf automake ];
+  buildInputs = [ libtool popt alsaLib ];
+  patchPhase = ''
+  substituteInPlace "src/main.cpp" --replace "/usr/share/pico/lang" "$out/share/lang"
+  echo "" > update_build_version.sh
+  '';
+  installPhase = ''
+    mkdir -p $out/bin
+    install -m 0755 nanotts $out/bin/
+    mkdir $out/share
+    cp -r $src/lang $out/share/lang
```

Comment:
```suggestion
    install -Dm644 -t $out/bin nanotts
    install -Dm644 -t $out/share lang
```

Reply:
shouldn't binary have `755` permission mode ?
or you were just suggesting to use the `install` command

Reply:
i edited the suggestion

Reply:
I've already pushed with that suggestion :)